.github/ISSUE_TEMPLATE/test-failure.yml

```diff
@@ -0,0 +1,51 @@
+name: Test Failure
+description: A test failure in CI
+labels: [">test-failure"]
+body:
+  - type: markdown
+    attributes:
+      value: Please fill out the following information, and ensure you have attempted to reproduce locally
+  - type: input
+    id: build_scan
+    attributes:
+      label: Build scan
+      description: A link to a Gradle buildscan with the failure
+    validations:
+      required: true
+  - type: input
+    id: repro_line
+    attributes:
+      label: Repro line
+      description: The reproduce line from the build output
+    validations:
+      required: true
+  - type: dropdown
+    id: reproducible
+    attributes:
+      label: Does it reproduce?
+      options:
+        - "Yes"
+        - "No"
```

Comment:
FWIW, for the Slack automation we also include a "Didn't try" option.

Reply:
Added